test/discrete/test_bdq.py

```diff
@@ -0,0 +1,144 @@
+import argparse
+import pprint
+
+import gym
+import numpy as np
+import torch
+
+from tianshou.data import Collector, VectorReplayBuffer
+from tianshou.env import DiscreteToContinuous, SubprocVectorEnv
+from tianshou.policy import BDQPolicy
+from tianshou.trainer import offpolicy_trainer
+from tianshou.utils.net.common import BDQNet
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    # task
+    parser.add_argument("--task", type=str, default="BipedalWalker-v3")
```

Comment:
This is definitely not a good test script. We'd like to run a toy environment (like cartpole or pendulum) to verify the algorithm works in a short amount of time (let's say less than 30s). BipedalWalker takes a long time to train and is not suitable for this case.